zerver/actions/message_send.py

```diff
@@ -1974,17 +1974,25 @@ def internal_send_stream_message_by_name(
     return sent_message_result.message_id
 
 
-def internal_send_huddle_message(
+def internal_prep_huddle_message(
     realm: Realm, sender: UserProfile, emails: List[str], content: str
-) -> Optional[int]:
+) -> Optional[SendMessageRequest]:
     addressee = Addressee.for_private(emails, realm)
-    message = _internal_prep_message(
+    return _internal_prep_message(
         realm=realm,
         sender=sender,
         addressee=addressee,
         content=content,
     )
+
+
+def internal_send_huddle_message(
+    realm: Realm, sender: UserProfile, emails: List[str], content: str
+) -> Optional[int]:
+    message = internal_prep_huddle_message(realm, sender, emails, content)
+
     if message is None:
         return None
+
```

Comment:
This seems like a fine refactor but usually I'd do it in a prep commit.

Reply:
Yeah, that's correct. I missed that. Done.